tools/docker-utils.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const tempy = require('tempy');
+const execa = require('execa');
+const { getUnleashServerVersion } = require('./unleash-server-utils');
+
+const tempDirPath = tempy.directory();
+
+/** @type {string} */
```

Comment:
IntelliJ yells at me because it thinks this is a Buffer (which is not true, as I pass `{ encoding: 'utf8' }`)